### PR TITLE
MRC-1778: Add endpoint for downloading summary report

### PR DESCRIPTION
**Pull request by r-ash**
This PR will
* add an endpoint `/download/summary` which streams bytes of a report generated as part of model run
* Update the pre-run stuff to work with this (also calibration data which I missed during last update)

### CODE REVIEW

File: inst/dummy_summary_report.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<ul>
```
Comment (richfitz) [inline inst/dummy_summary_report.html:1]: disappointed in your choice of ipsum :joy:

File: tests/testthat/test-api.R [hunk 2]
```diff
@@ -945,6 +945,83 @@ test_that("api endpoint_download_coarse_output_head returns headers only", {
...
+
+  expect_equal(response$status_code, 200)
+  expect_match(
+    response$headers$`Content-Disposition`,
+    'attachment; filename="MWI_\\d+-\\d+_summary_report.zip"')
+  size <- length(response$data)
+  expect_equal(response$headers$`Content-Length`, size)
+  expect_equal(size, file.size(system_file("dummy_summary_report.html")))
+
+  ## HEAD endpoint
```
Comment (richfitz) [inline tests/testthat/test-api.R:970]: This looks like it's testing the wrong thing? (coarse inputs vs summary)

**Review by richfitz — CHANGES_REQUESTED** [review]
Looks like one incomplete test, but otherwise LGTM

File: tests/testthat/test-api.R (hunk 2 above)
Comment (r-ash) [reply tests/testthat/test-api.R:971]: Thanks, updated that

File: inst/dummy_summary_report.html (hunk 1 above)
Comment (r-ash) [reply inst/dummy_summary_report.html]: Consider it gone!

Comment (richfitz) [reply inst/dummy_summary_report.html]: What is this using now? I just figured you might use a more esoteric placeholder text

**Review by richfitz — APPROVED** [review]

**Pull request merged**